### PR TITLE
[docs] update to expose libraries + landing page

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

This reorganizes the tune docs to give important algorithms their own page (we should add more as new algorithms are added). This should hopefully help with SEO -- currently if you search for "HyperBand" the tune docs don't even show up in the first couple pages. However if you search for "A3C" or "Rllib" those terms already rank highly.

lbs

### CODE REVIEW

**Comment by richardliaw** [comment]
Somehow it might be nice to indicate that HyperBand isn't a library, as with PBT. I do believe these things are worth highlighting and am not sure about the right formatting.

@robertnishihara @pcmoritz any thoughts?

**Comment by ericl** [comment]
Yeah I don't know if readthedocs has some syntax for a nested page.

**Comment by richardliaw** [comment]
screenshot 2018-03-06 17 48 13

**Comment by robertnishihara** [comment]
seems good to me

**Review by richardliaw — APPROVED** [review]
@ericl, should do a read over my changes before merge

File: doc/source/hyperband.rst
Comment (ericl) [inline doc/source/hyperband.rst]: Should we recommend this as the default one?

File: doc/source/hyperband.rst
Comment (ericl) [inline doc/source/hyperband.rst]: This phrasing sounds more like a blog post, reword for documentation?

**Review by ericl — COMMENTED** [review]
Looks good, couple comments on phrasing

**Comment by richardliaw** [comment]
addressed comments and tweaked a little more; I'll merge for now after tests pass(as there are no substantial changes) but if there are issues we can just patch it.

**Pull request merged**
